packages/python-django/python-django.spec

```diff
@@ -6,8 +6,8 @@
 %global srcname django
 
 Name:           %{?scl_prefix}python-%{srcname}
-Version:        3.2.12
-Release:        2%{?dist}
+Version:        3.2.13
+Release:        1%{?dist}
```

Comment:
this release already exists: http://koji.katello.org/koji/buildinfo?buildID=70459

Reply:
So here's the problem, If I cherry pick and later bump in the same PR, our rpmcheck will fail, let me cherry-pick the change, and after the merge I'll bump the release to build against 3.9